unstructured/partition/lang.py

```diff
@@ -17,3 +156,55 @@ def convert_old_ocr_languages_to_languages(ocr_languages: str):
     """
 
     return ocr_languages.split("+")
+
+
+@requires_dependencies("pytesseract")
+def convert_language_to_tesseract(lang: str) -> str:
+    """
+    Convert a language code to its tesseract formatted and recognized langcode(s), if supported.
+    """
+    # if language is already tesseract langcode, return it immediately
+    # this will catch the tesseract special cases equ and osd
+    # NOTE(shreya): this may catch some of the cases of choosing between a plain vs suffixed tesseract code
+    if lang in PYTESSERACT_LANGS:
+        return lang
+
+    # get iso639 language object
+    try:
+        lang_iso639 = iso639.Language.match(lang)
+    except:
+        logger.warning(f"{lang} is not a valid standard language code.")
+        return ""
```

Comment:
did you test this return value? I think this will still add to our string in the `join` call where we're calling this function. Unless tesseract will just process the extra '+' signs and go to the next lang value?

```
In [37]: amanda = ["", "rye", "amanda", "", "test"]

In [38]: "+".join(amanda)
Out[38]: '+rye+amanda++test'
```

Reply:
good catch! I added a filter for empty strings in the calling function now